content/millennium/dstu2/general-clinical/procedure.md

```diff
@@ -43,18 +43,19 @@ _Implementation Notes_
 
 ### Parameters
 
- Name         | Required?                             | Type          | Description
---------------|---------------------------------------|---------------|------------------------------------------------------------------------------------
- `_id`        | This, or one of `patient` or `subject`| [`token`]     | The logical resource id associated with the resource. Example: `_id=7891`
- `patient`    | This, or one of `_id` or `subject`    | [`reference`] | The patient subject of the Procedure. Example: `patient=12345`
- `subject`    | This, or one of `_id` or `patient`    | [`reference`] | The subject of the Procedure. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/12345` or `subject:Patient=12345`
- `date`       | N                                     | [`date`]      | The date/time when the Procedure was performed. Must use the `ge` and/or `le` prefixes. Example: `date=le2017-02-01T10:30:00Z`
+ Name              | Required?     | Type          | Description
+-------------------|---------------|---------------|-------------------------------------------------------------------------------------
+`_id`              | Conditionally | [`token`]     | The logical resource id associated with the resource. Example: `_id=7891`
+`patient`          | Conditionally | [`reference`] | The patient of the Procedure. Example: `patient=12345`
+`subject`          | Conditionally | [`reference`] | The subject of the Procedure. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/12345` or `subject:Patient=12345`
+`date`             | No            | [`dateTime`]  | The date/time when the Procedure's `performedDateTime` was performed. Must use the `ge` and/or `le` prefixes. Example: `date=le2017-02-01T10:30:00Z`
 
 Notes:
 
-  - The `_id` parameter may not be provided at the same time as the `patient`, `subject`, or `date` parameters.
-
-  - The `date` parameter must have a time, may be provided up to two times, and must use the `ge` or `le` prefixes. When provided twice, the lower value must have the `ge` prefix and the higher value must have the `le` prefix.
+* The `_id` parameter may not be provided at the same time as the `patient`, `subject`, or `date` parameters.
```

Comment:
Drop the conditional requirement line here, and use the descriptions in the R4 resource's table over on this DSTU2 resource, please.  Your fixes in the R4 file match what the technical writers are looking for.